src/redux/actions.js

```diff
@@ -509,6 +509,12 @@ export function setModificationsInProgress(isModificationsInProgress) {
     };
 }
 
+export const STUDY_DISPLAY_MODE = {
```

Comment:
is this needed ?

Reply:
has been included by mistake during the merge with main